app/src/main/java/com/kickstarter/ui/views/CommentCard.kt

```diff
@@ -129,20 +133,29 @@ class CommentCard @JvmOverloads constructor(
         binding.commentBody.setTextColor(ContextCompat.getColor(context, commentBodyTextColor))
     }
 
+    private fun shouldShowReplyButton(cardCommentStatus: CommentCardStatus) =
```

Comment:
@hadia for the new states when dealing with errors, just add the new ones in this function. It should make solving conflicts easier

Reply:
Ties back into my previous comment, the card status is for the replies, not the reply button. Unless I'm missing something here 😅